### ISSUE TITLE
winerror 10061

### ISSUE THREAD

**Issue by nweko** [1Euro7Cent/Mrballou-drawbot#45]
i keep getting this message over and over
"Starting
establishing connection with server on port 25600
Got error [WinError 10061] No connection could be made because the target machine actively refused it
No connection to server
trying to reconnect..."

so i used to use this and ive never had this issue before so im not sure how to fix

**Comment by 1Euro7Cent** [comment]
can you try to change the config's port to something diffrent then 25600. for eg 25601

**Comment by nweko** [comment]
yes, i changed it a few times and i still get the error no matter which port it is

**Comment by SpeezyBeezy** [comment]
The same issue here changed the port in config.json, no luck
![2024-12-30 20-42-41](https://github.com/user-attachments/assets/bb4d781f-5a2c-4890-afe5-47eb94c2be85)

The same WinError 10061 error message
and a little GUI window pops up which just says "No connection to server. Reconnecting... Try 25 Is the server running?"
Tried running with Admin privileges didn't do the trick either

**Comment by 1Euro7Cent** [comment]
Judging by your screenshot, I cannot see the servers console. Is it running? (drawbot.exe or index.js)

**Comment by SpeezyBeezy** [comment]
Yes that seems to be the problem
![image](https://github.com/user-attachments/assets/c0495170-baba-4944-a679-d377db2b429c)
When I open drawbot.exe this terminal window appears for like a split second and then closes
First opened gui.exe and then drawbot.exe or vise versa nither works

And then I tried with the start.bat file which didn't work either but the terminal didn't close and it had a error code
![image](https://github.com/user-attachments/assets/812f5159-29f0-45b4-ad66-35988e13ff28)
Seems like the problem has something to do with the saves.json file?
![image](https://github.com/user-attachments/assets/2db416b0-25a7-4125-a5d2-c463027b60fc)
Which is empty

**Comment by 1Euro7Cent** [comment]
Try updating the bot (just the drawbot.exe should be enough) [here](https://github.com/1Euro7Cent/Mrballou-drawbot/releases/tag/3.20.3)

**Comment by SpeezyBeezy** [comment]
Sorry for the late reply
It works just fine now, Thanks for the fix!
